altapay.php

```diff
@@ -5,24 +5,32 @@
  * Description: Payment Gateway to use with WordPress WooCommerce
  * Author: AltaPay
  * Author URI: https://www.altapay.com/knowledge-base/omni-channel/
- * Version: 3.1.1
+ * Version: 3.2.0
  * Name: SDM_Altapay
  * WC requires at least: 3.0.0
- * WC tested up to: 4.7.0
+ * WC tested up to: 4.8.0
  *
  * @package Altapay
  */
 
 use Altapay\Classes\Core;
 use Altapay\Classes\Util;
 use Altapay\Helpers;
+use Altapay\Api\Payments\CaptureReservation;
+use Altapay\Exceptions\ResponseHeaderException;
+use Altapay\Response\CaptureReservationResponse;
+use Altapay\Api\Payments\RefundCapturedReservation;
+use Altapay\Api\Payments\ReleaseReservation;
+use Altapay\Response\ReleaseReservationResponse;
+use Altapay\Api\Others\Payments;
 
 if ( ! defined( 'ABSPATH' ) ) {
 	exit; // Exit if accessed directly
 }
 
 // Include the autoloader so we can dynamically include the rest of the classes.
 require_once __DIR__ . DIRECTORY_SEPARATOR . 'includes/Autoloader.php';
```

Comment:
Since we have defined the autoloade section in composer.json it should be possible to just use the vendor autoloader and not both.